src/fixate/drivers/dmm/fluke_8846a.py

```diff
@@ -62,6 +62,7 @@ def __init__(self, instrument, *args, **kwargs):
         self._mode = None
         # Set to True to have explicit error checks on each read
         self.legacy_mode = False
+        self.serial = self.get_serial()
```

Comment:
I would leave this out. We're not likely to call get_serial often, so if we do need it more than once it's not too big a cost to call out to the instrument multiple times.